frontend/.env.example

```diff
@@ -2,6 +2,16 @@
 # See https://chainlist.org/
 REACT_APP_SUPPORTED_CHAIN_ID="0x89"
 REACT_APP_SUPPORTED_CHAIN_NAME="Polygon"
+REACT_APP_RPC_URL="https://rpc-mainnet.maticvigil.com"
+# multiple urls can be used for backups, simply separate them with a comma
+REACT_APP_RPC_URLS="https://rpc-mainnet.maticvigil.com"
+REACT_APP_BLOCK_EXPLORER_URLS="https://polygonscan.com/"
+REACT_APP_CHAIN_ICON_URLS="https://polygonscan.com/images/svg/brands/polygon.svg?v=1.3"
+REACT_APP_SUPPORTED_CHAIN_NATIVE_CURRENCY_NAME="MATIC"
```

Comment:
Should these include the word `DEFAULT`? Might be easier to read.

`REACT_APP_DEFAULT_CURRENCY_NAME`